### DISCUSSION TITLE
Need help with implementation

### DISCUSSION THREAD

**Discussion by adamgajos** [deepgram/community#246] [General help] [answered]
Hi, I'm pretty new to web development and I'd like to create a web app using python as a backend and Deepgram API. 
I'd wish to have a feature where user could talk and the speech would be recognised as they speak in real-time using Deepgram and in the same time it would be displayed on the screen. Is it even possible? If so, how can I approach it?
Thanks in advance, cheers!

**Comment by shirgoldbird** [comment, marked as answer]
Hey @adamgajos! Take a look at this [tutorial](https://deepgram.com/learn/live-transcription-flask) for integrating Deepgram with the Python library Flask to build a web app that transcribes the user's microphone. Hope this helps :)

**Reply by jpvajda** [reply]
@adamgajos It looks like that link @shirgoldbird  provided leads to a 404, so I think you can find what you need here now: https://deepgram.com/learn/live-transcription-flask

We recently redid our Blog so some of the URLs aren't redirecting correctly.

**Reply by shirgoldbird** [reply]
Thanks, @jpvajda! I've updated the link in my comment as well. 

I'll mark this as answered for now, but @adamgajos, feel free to let us know if you have any other questions.
